<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowCountries</class>
 <widget class="QMainWindow" name="ShowCountries">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>197</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelCountryName">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Landnavn</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFirst">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;&lt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelLandID">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>LandID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNext">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>150</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLast">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>150</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPrev">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditCountryName">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxLandID">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>79</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelInfo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Her kan du se de ulike statusene som er registrert i systemet. Vil du endre på ID-en til en status, må det gjøres i kommandolinja.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
